core/arch/x86_64/kernel/exceptions.S

```diff
@@ -0,0 +1,174 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2009 Corey Tabaka
+ * Copyright (c) 2016 Travis Geiselbrecht
+ * Copyright (c) 2018 Intel Corporation
+ */
+
+#include <asm.h>
+#include <descriptor.h>
+
+#define NUM_INT 0x100
+#define NUM_EXC 0x14
+
+.text
+
+/* interrupt service routine stubs */
+_isr:
+.set i, 0
+.rept NUM_INT
```

Comment:
you can check the code in evmm, it uses ".align 16" to make code simpler

Reply:
code updated according to comment, please review